tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -1479,4 +1479,29 @@ class CompletionSuite extends BaseCompletionSuite {
     "",
   )
 
+  check(
+    "pkg",
+    s"""|object Foo {
+        |  scala.coll@@
```

Comment:
perhaps, it would be better if the test cases cover the cases listed [here](https://github.com/scalameta/metals/pull/4223/files#r936944963)

Reply:
only the last one is covered now

Reply:
They should cover all of them now